src/Model/BaseManager.php

```diff
@@ -149,8 +149,21 @@ public function getTableName()
         return $metadata->table['name'];
     }
 
+    /**
+     * NEXT_MAJOR: Remove $objectName parameter along with psalm and phpstan suppresions.
```

Comment:
```suggestion
     * NEXT_MAJOR: Remove $objectName argument along with psalm and phpstan suppressions.
```

Reply:
I've left argument and parameter since `$objectName` should be removed from the method definition and from the `ObjectManager::clear()` method call.